pkg/releasetesting/environment.go

```diff
@@ -37,7 +37,7 @@ type Environment struct {
 	Stream      services.ReleaseService_RunReleaseTestServer
 	Timeout     int64
 	Parallel    bool
-	Parallelism uint32
+	Parallelism int64
```

Comment:
is there a way we can revert this? This would be considered a backwards compatibility breaker.

Reply:
Good point, I'll revert back to uint32

Reply:
Its back to uint32 now along with the other variables